### DISCUSSION TITLE
403 when adding units

### DISCUSSION THREAD

**Discussion by mateusz-olszak** [WeblateOrg/weblate#8227] [General]
Hi,
I'm trying to add new unit translation via rest api but I receive status 403 and info can not add unit.

I do `POST /api/translations/(string: project)/(string: component)/(string: language)/units/` with my token. and body:

`{
 "key": "somekey",
 "value": ["some value"]
}`

I can add translation unit via admin panel but when i try do this programatically i get response as described.
Could someone help me to resolve this issue ?

**Comment by nijel** [comment]
The response will contain some details as well. The supported operations depend on the component configuration.
